package.json

```diff
@@ -9,12 +9,12 @@
   "scripts": {
     "postinstall": "ngcc --tsconfig './tsconfig.app.json' --use-program-dependencies && ngcc --source 'node_modules/@angular'",
     "ng": "ng",
-    "start": "ng serve",
-    "build": "ng build --prod",
+    "start": "NG_PERSISTENT_BUILD_CACHE=1 ng serve",
+    "build": "NG_PERSISTENT_BUILD_CACHE=1 ng build --configuration production",
```

Comment:
for local builds, we are not enabling prod config yet, right?

Reply:
Removed